allen-cell-animated/cell-feature-explorer
# Feature/sort megacards from newest to oldest

Problem
=======
Resolves: [Sort megacards from newest to oldest](https://aicsjira.corp.alleninstitute.org/browse/CFE-141)

Solution
========
@meganrm has added a `dateCreated` timestamp for each megaset in the test database, so the megasets are now sorted from newest to oldest before being rendered as megacards. (The plan is to modify the Firebase scripts soon so that the timestamps are added automatically for newly uploaded datasets)

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)
* This change requires updated or new tests

Change summary:
---------------
* Wrote new sorting function and test
* Removed old sorting function and tests

Steps to Verify:
----------------
1. Pull this branch and `npm run start:test-db`
2. Verify that the datasets are in the order we want, as shown in the screenshot below

Screenshots (optional):
-----------------------
<img width="970" alt="image" src="https://user-images.githubusercontent.com/12690133/153521759-094295d9-f76c-4f57-b0e5-07c61ad69c24.png">

Thanks for contributing!
